pyecharts/base.py

```diff
@@ -140,36 +139,21 @@ def cast(seq):
 
     def _repr_html_(self):
         """ 渲染配置项并将图形显示在 notebook 中
+        chart/page => charts
+        chart.js_dependencies => require_config => config_items, libraries
+        :return A unicode string.
         """
-        dom = self._render_notebook_dom_()
-        component = self._render_notebook_component_()
         require_config = CURRENT_CONFIG.produce_require_configuration(
-            self._js_dependencies
+            self.js_dependencies
+        )
+        config_items = require_config['config_items']
+        libraries = require_config['libraries']
+        env = engine.create_default_environment()
+        return env.render_chart_to_notebook(
+            charts=(self,),
+            config_items=config_items,
```

Comment:
why not just pass on : **config_items

Reply:
For readability.
In the template file , *config_items* and *libraries* are used directly instead of *require_config*
```
    require.config({
        paths: {
            {{ config_items | join(', ') }}
        }
    });
....
    require([{{ libraries | join(', ') }}], function(echarts) {
        {{ echarts_js_content_wrap(*charts) }}
    });
```